### ISSUE TITLE
Use z coordinate to get 3D prediction

### ISSUE THREAD

**Issue by abdou31** [yinguobing/cnn-facial-landmark#68]
Hello Yin,
I would like to use z coordinate and add it to the pts file.
Is it possible to train dataset (images + .json files ) with x, y and z coordinates with the CNN that you created?
For e.g, if landmarks that I want to use are 40 landmarks,
units in logits layer should equal to 120.

**Comment by yinguobing** [comment]
I think it's possible, and in that case the number of node in the output layer is 120 as you said.

**Comment by abdou31** [comment]
So , pts file should be in this form :
```
n_points {
9.14523 5.2145 1.365
1.2541 1.254 1.879
}
```

But what should I change in pts_tools.py and extract_points_from_ibug.py?
